src/components/images/imageLoader.js

```diff
@@ -124,15 +124,18 @@ import 'css!./style';
 
     export function lazyChildren(elem) {
         if (userSettings.enableBlurhash()) {
-            for (const lazyElem of elem.querySelectorAll('.lazy')) {
+            const lazyElems = Array.from(elem.querySelectorAll('.lazy'));
+
+            lazyElems.forEach((lazyElem) => {
```

Comment:
What is wrong with `for..of`?

Reply:
i don't see any. Changed for that syntax. Please check if that's ok.